assets/css/style.css

```diff
@@ -0,0 +1,223 @@
+body {
```

Comment:
You did really well so I want to give you an advanced tip. Use variables for repeated properties.
https://www.w3schools.com/css/css3_variables.asp
```
:root {
  --main-color: rgb(26, 25, 25);
}
```

then you can use it later with:
```css
.style {
  background-color: var(--main-color);
}
```